qgroundcontrol.pro

```diff
@@ -86,17 +86,34 @@ exists(user_config.pri):infile(user_config.pri, CONFIG) {
 # is present. It's useful to run "regular" builds to make sure you didn't
 # break anything.
 
+CUSTOM_DIR = $$PWD/qgc-custom
+
 contains (CONFIG, QGC_DISABLE_CUSTOM_BUILD) {
     message("Disable custom build override")
 } else {
-    exists($$PWD/custom/custom.pri) {
-        message("Found custom build")
+    CUSTOM_PRI = $$PWD/qgc-custom/custom.pri
```

Comment:
This breaks/changes existing behavior for all current users of custom builds. In place custom dir should stay named 'custom' to not break existing users.